docs/getting-started/supported-languages-and-tools.md

```diff
@@ -99,6 +109,7 @@ The table below lists all languages and frameworks that Codacy supports and the
       <td><a href="https://github.com/SonarSource/sonar-dotnet">SonarC#</a></td>
       <td>-</td>
       <td><a href="https://trivy.dev">Trivy</a></td>
+      <td><a href="https://trivy.dev">Trivy</a> (scans <code>packages.lock.json</code>, <code>packages.config</code>, <code>.deps.json</code>) <a href="#any-path"><sup>3</sup></a></td><!-- TODO TAROT-2398 confirm that these file formats match what we analyze -->
```

Comment:
Does Trivy scan all these?

Reply:
Sorry, it's best to look at this table, 'Filesystem' column -- https://aquasecurity.github.io/trivy/v0.45/docs/coverage/language/#supported-languages